data/secscan_model/datatypes.py

```diff
@@ -29,6 +29,9 @@ class ScanLookupStatus(IntEnum):
     "Vulnerability",
     ["Severity", "NamespaceName", "Link", "FixedBy", "Description", "Name", "Metadata"],
 )
+Metadata = namedtuple(
+    "Metadata", ["UpdatedBy", "RepoName", "RepoLink", "DistroName", "DistroVersion"]
```

Comment:
@arajkumar is there any advantage in not using the same field names as the vulnerability report? "Updater" instead of "UpdatedBy"?

Reply:
No specific reason for `UpdatedBy` field, but other fields has to have prefix(Repo & Distro) to avoid nested properties.